### PR TITLE
Hotfix/1.31.7 2

### PR DESCRIPTION
**Pull request by 1saf**
# Description

- Fixes flickering similar pools card on the first step of pool creation
- Fixes the issue with accordion position not updating correctly on mobile when alerts are mounted/unmounted in pool creation
- Fixes no spacing issue on the bottom of first step of pool creation where 8 tokens have been added

Fixes # (issue)

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- For accordion testing, switch to mobile and make sure to toggle all options which may trigger an alert or change the height of the card at each step. The accordion should animate to be under the main card at all times.
- Add 8 tokens to the first step - see if there is ample space for wiggling at the bottom of the page
- Similar pools card should not flash momentarily on first visit of the pool creation page.

## Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code where relevant, particularly in hard-to-understand areas
- [X] My changes generate no new console warnings
- [X] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

**Review by garethfuller — APPROVED** [review]
Spacing at bottom looks good now, and issue with spacing between main card and accordion on mobile is fixed. However, I'm still seeing the issue with the similar pools card flickering on first-page entry.

Also, after adding a bunch of tokens and then removing them I see this weird spacing:
Screenshot 2021-12-06 at 11 26 16

Please wait for the weekly release to be merged and then update the version to 1.32.1.

File: src/components/animate/AnimatePresence.vue [hunk 1]
```diff
@@ -75,13 +75,18 @@ export default defineComponent({
       // on mount we set initial values, but the issue is that enter will run at
       // the same time, setTimeout(0) makes the animation run on the next
       // available tick, so it's instant visually but on a tick delay for code
+      emit('on-presence', { isCompleted: false });
```
Comment (timjrobinson) [inline src/components/animate/AnimatePresence.vue:78]: What is this `isCompleted`? It doesn't seem to be used in the functions called from this emit.

File: src/components/animate/AnimatePresence.vue [hunk 2]
```diff
@@ -75,13 +75,18 @@ export default defineComponent({
...
       setTimeout(
         () =>
           anime({
             targets: el,
             ...props.animate,
             easing: 'spring(0.2, 80, 10, 0)',
-            complete: () => done()
+            complete: () => {
+              done();
+              emit('on-presence', { isCompleted: false });
```
Comment (timjrobinson) [inline src/components/animate/AnimatePresence.vue:88]: isCompleted is false in the complete callback?

**Comment by 1saf** [comment]
> Spacing at bottom looks good now, and issue with spacing between main card and accordion on mobile is fixed. However, I'm still seeing the issue with the similar pools card flickering on first-page entry.
> 
> Also, after adding a bunch of tokens and then removing them I see this weird spacing: Screenshot 2021-12-06 at 11 26 16
> 
> Please wait for the weekly release to be merged and then update the version to 1.32.1.

unable to recreate the spacing issue

**Comment by garethfuller** [comment]
@1saf you'll need to update the version again.

**Pull request merged**